app/views/account/applications/index.html.erb

```diff
@@ -16,11 +16,11 @@
    })
 %>
 
-<% if flash[:application_id] %>
+<% if flash[:success_alert] %>
   <% content_for(:custom_alerts) do %>
     <%= render "govuk_publishing_components/components/success_alert", {
-        message: "Permissions updated",
-        description: permissions_updated_description(flash[:application_id]),
+        message: flash[:success_alert]["message"],
+        description:flash[:success_alert]["description"].html_safe,
```

Comment:
Could we call `.html_safe` in the helper or the controller, which would then allow us to simplify this hash to `flash[:success_alert]`, or do you prefer to have all the options written out in the view?

Reply:
I'm afraid we can't call `html_safe` in the helper/controller as the escaping is stripped out when it's serialized as JSON when stored as a browser cookie. [1]

This same serialization is what converts symbols to strings, meaning we also can't pass `flash[:success_alert]` to the Success Alert, as the `message` and `description` keys are converted to strings to store in the cookie, while the component expected an object with symbol keys.[2]

[1]: https://groups.google.com/g/rubyonrails-core/c/z52zgDgUmbs
[2]: bigbinary.com/blog/flash-access-changes-rails-4-1

Reply:
I've also updated this to `sanitize` as [apparently](https://apidock.com/rails/String/html_safe) it's recommended over `html_safe`